make-docs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+# Execute from the repo's root directory
+rm -rf docs/source/*.txt
+SPHINX_APIDOC_OPTIONS=members,undoc-members,show-inheritance sphinx-apidoc -eM -s txt -o docs/source/ exa *test*
+cd docs
+make html
```

Comment:
Is it a good practice to add a symlink pointing to the `build/index.html dir`?
I'm doing it on a repository I am working on and it works
We would probably have to add the index.html to the gitignore.

Reply:
Can you say more about this? I'm not sure what you mean about symlinks.

Reply:
I just mean to make a soft link to the inde.html file with `ln -s build/html/index.html index.html`.
Also, I'm guessing this assumes that any developer working on this will be using WSL on windows?

Reply:
We could make that symlink for ease of local access. I think you're right on the second - I'll add a .bat file for native Win ad-hoc docs compilation.

Reply:
The first point was more of a question if it'd be good practice to do that.

I think if we address the second point for windows users this would be fine to merge.